src/src/WPCOTool/Admin/Settings.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace WPCOTool\Admin;
+
+/**
+ * Class Settings
+ * Manage plugin settings screens and admin menu
+ * @package WPCOTool\Admin
+ */
+class Settings {
+
+    /**
+     * Plugin title
+     */
+    private $title;
+
+    /**
+     * Required capability
+     * @var string
+     */
+    private $capability = 'administrator';
```

Comment:
Let's use the admin capability `manage_options` rather than `administrator` role.

Reply:
Created issue #17 for this.